doc/nrf/includes/advanced_conf_matter.txt

```diff
@@ -3,6 +3,8 @@ Advanced configuration options
 
 This section describes other configuration options for the sample.
 
+.. _advanced_conf_matter_dfu:
```

Comment:
This breaks the doc build. I don't think we need this.

Reply:
I wanted to link to this section from the `Matter template using only internal memory`. Do you think it's not needed? Is there other way to do that?

Reply:
Try renaming it, at least.

Reply:
But what is the issue? This name is unique and I refer to that in a single place, so I don't quite get where the duplication error comes from.

Reply:
You can try to drop that link tag and instead, link directly to the heading like this: :ref:`Device Firmware Upgrade support`.

Reply:
Still no luck with linking. Trying to figure out how we can link to the reused file and specifically to that section within.

Reply:
I'm afraid we have to drop the link for now, and as a temporary solution, just mention the heading to make the PR pass the doc build.

Reply:
Ok, I've removed the link and fixed the issues found by Damian. Please re-check once again.